activestorage/app/controllers/active_storage/base_controller.rb

```diff
@@ -2,18 +2,9 @@
 
 # The base class for all Active Storage controllers.
 class ActiveStorage::BaseController < ActionController::Base
-  include ActiveStorage::SetCurrent
+  include SetCurrent, Streaming
```

Comment:
Looks like the tests are failing because `SetCurrent` can't be resolved from within `ActiveStorage::BaseController`. I think Ruby 2.4/2.5 removed that resolution.